docs/modules/ROOT/partials/nav-howtos.adoc

```diff
@@ -1,7 +1,15 @@
-* xref:app-catalog:ROOT:how-tos/implement_new_service_offering.adoc[Implement New Service Offering]
-
 * Crossplane
-** xref:app-catalog:ROOT:how-tos/crossplane/investigate_service_instances.adoc[Investigate Service]
+** xref:app-catalog:ROOT:how-tos/crossplane/investigate_service_instances.adoc[Investigate a Service]
+** xref:app-catalog:ROOT:how-tos/crossplane/implement_new_service_offering.adoc[Implement a New Service]
+
+* Crossplane Service Broker
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/overview.adoc[Overview]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/setup_crossplane_service_broker.adoc[Setup a _Crossplane Service Broker_]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/setup_service_catalog.adoc[Setup a _Service Catalog_]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/bearer_token_authentication.adoc[HTTP _Bearer Token_ authentication]
+** xref:app-catalog:ROOT:how-tos/crossplane_service_broker/connect_service_catalog_to_service_broker.adoc[Connect the _Service Catalog_ to the _Service Broker_]
```

Comment:
I would move this page to the beginning of this list as I find this to be the most important one.

Reply:
Since antorra generates such nice "Prev" and "Next" pointers at the end of the how-to (see screenshot below), I figured it would be best to arrange the how-tos in the order in which I had them originally, as they (at least slightly) build upon eachother.


---

<img width="862" alt="grafik" src="https://user-images.githubusercontent.com/804532/122187014-ece46100-ce8e-11eb-8602-dddb325e53c9.png">

---